### PR TITLE
add 70 climbing-stairs

### PR DESCRIPTION
**Pull request by kobakobashu**
time : 15m

### CODE REVIEW

**Comment by kobakobashu** [comment]
Please check my code.

File: 70.climbing-stairs.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+#
+
+# @lc code=start
+class Solution:
+    def climbStairs(self, n: int) -> int:
+        """
+        time: O(n)
+        space: O(n)
+        """
+        step = [0]*(n+2)
```
Comment (ryuji0123) [inline 70.climbing-stairs.py:14]: step = [i if i <= 2 else 0 for i in range(n + 2)]

File: 70.climbing-stairs.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        step[0] = 0
+        step[1] = 1
+        step[2] = 2
+        if 3 <= n:
+            for step_num in range(3,n+1):
```
Comment (ryuji0123) [inline 70.climbing-stairs.py:19]: for num_step in range(3,n+1):

**Review by ryuji0123 — CHANGES_REQUESTED** [review]

**Comment by kobakobashu** [comment]
I fixed my code.  Please check it.

**Review by ryuji0123 — APPROVED** [review]
LGTM

**Pull request merged**